### PR TITLE
Bug 1519995 - Convert Graphs view to React - test cards section

### PR DESCRIPTION
**Pull request by sarah-clements**
This is for the divs/cards containing tests that are selected from the `TestDataModal` (when you click the 'add more test data' button).
Screen Shot 2019-06-25 at 3 01 07 PM

Edit: I've decided to move the changes to the `TestDataModal` to a separate pr since it contained a fix to the filter behavior.

### CODE REVIEW

File: ui/js/controllers/perf/graphs.js [hunk 1]
```diff
@@ -35,9 +35,8 @@ perf.controller('GraphsCtrl', [
     '$window', '$q', '$timeout',
     function GraphsCtrl($state, $stateParams, $scope, $rootScope,
         $uibModal, $window, $q, $timeout) {
-        var availableColors = ['maroon', 'navy', 'pink', 'turquoise', 'brown',
-            'red', 'green', 'blue', 'orange', 'purple'];
-
+        var availableColors = ['darkseagreen', 'lightseagreen', 'darkslateblue', 'darkgreen', 'steelblue', 'darkorchid', 'blue', 'darkcyan'];
```
Comment (camd) [inline ui/js/controllers/perf/graphs.js]: could this be a ``const`` instead of a ``var``?

File: ui/perfherder/graphs/SelectedTestsContainer.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+import perf from '../../js/perf';
+
+import TestCard from './TestCard';
+
+const SelectedTestsContainer = props => {
+  // TODO seriesList is the same as testsDisplayed in TestDataModel - change
+  // name to keep it consistent
+  const { seriesList } = props;
+  return (
```
Comment (camd) [inline ui/perfherder/graphs/SelectedTestsContainer.jsx]: I usually try to follow this eslint rule, even though we don't have it activated: https://eslint.org/docs/rules/newline-after-var

Up to you, though.  :)

File: ui/js/controllers/perf/graphs.js (hunk 1 above)
Comment (sarah-clements) [reply ui/js/controllers/perf/graphs.js]: yup, I'll change it.

File: ui/perfherder/graphs/SelectedTestsContainer.jsx (hunk 2 above)
Comment (sarah-clements) [reply ui/perfherder/graphs/SelectedTestsContainer.jsx]: That's a great rule - thanks!

**Pull request merged**